src/WebJobs.Extensions.EasyTables/Config/MobileServiceApiKeyHandler.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.EasyTables
+{
+    /// <summary>
+    /// A <see cref="DelegatingHandler"/> that adds a ZUMO-API-KEY header to every request.
+    /// </summary>
+    internal class MobileServiceApiKeyHandler : DelegatingHandler
```

Comment:
Hmmm, this isn't an inherent capability of MobileServiceClient? I.e., set it's key and it will do the right thing?